src/resources/help/en_US/relnotes3.3.0.html

```diff
@@ -33,7 +33,7 @@
 <p><b><font size="4">General features added in version 3.3</font></b></p>
 <ul>
   <li>Updated the ORNL host file list to include only current machines.</li>
-  <li>General Feature 2</li>
+  <li>After a successful Export databses, the informational message now contains the host and path to the export.</li>
```

Comment:
remove `s` at end of `databases`

Reply:
Actually, an "a" is also missing from "databses". So change "databses" to "database".